src/core/task_mngr.c

```diff
@@ -0,0 +1,195 @@
+#include "task_mngr.h"
+
+
+//this functions are inline because they are run only from task manager at one place.
```

Comment:
Another typo. Should be `These functions`.
Also, the comment does not seem correct. Am I missing something or is the `inline` keyword missing?